sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -805,6 +805,22 @@ abstract class CSVSuite
     }
   }
 
+  test("SPARK-37575: null values should not reflect to any characters by default") {
+    val litNull: String = null
```

Comment:
`litNull` is used only once. Let's inline it.

Reply:
I added some new codes and now `litNull` is used several times.